Content.Server/Theta/ShipEvent/Systems/ShipEventFactionSystem.cs

```diff
@@ -176,7 +174,7 @@ private void OnTeammateSpeak(EntityUid uid, ShipEventFactionMarkerComponent comp
         if (session == null)
             return;
 
-        var chatMsg = session.ConnectedClient.UserName + " : " + args.Message;
+        var chatMsg = "\\[Team\\] " + session.ConnectedClient.UserName + ": " + args.Message;
```

Comment:
```suggestion
        var chatMsg = "[Team] " + session.ConnectedClient.UserName + ": " + args.Message;
```
Ну и как бы, а зачем такое экранирование то?

Reply:
Потому что то, что в квадратных скобках - не отображается в чате, даже с "".